src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public abstract class Figure {
+    private String color;
+
+    public Figure() {
+        ColorSupplier cc = new ColorSupplier();
+        this.color = cc.getRandomColor();
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    abstract String getFigureInfo();

```

Comment:
let's create a separate interface Drawable and move this method there and implement it here, also rename it to `draw()` and make it void

Reply:
What about this comment?